Chapter09/recipe-06/cxx-example/account/account.h

```diff
@@ -1,4 +1,5 @@
-#pragma once
+#ifndef ACCOUNT_H_INCLUDED
```

Comment:
Why remove `#pragma once`?

Reply:
Because CFFI chokes on it and gives a warning.

Reply:
Can you add a comment about that? :) Just so I don't forget and go back to `#pragma once`